extensions/replication/queueProcessor/task.js

```diff
@@ -8,11 +8,19 @@ const config = require('../../../conf/Config');
 const zkConfig = config.zookeeper;
 const repConfig = config.extensions.replication;
 const sourceConfig = repConfig.source;
-const destConfig = repConfig.destination;
 
+const site = process.argv[2];
+assert(site, 'QueueProcessor task must be started with a site as argument');
+
+const bootstrapList = repConfig.destination.bootstrapList
+    .filter(item => item.site === site);
+assert(bootstrapList.length === 1, 'Invalid site argument. Site must match ' +
+    'one of the replication endpoints defined');
+
+const destConfig = Object.assign({ bootstrapList }, {}, repConfig.destination);
```

Comment:
Testing this out locally, I'm seeing  `destConfig.bootstrapList` being set to all items in `repConfig.destination.bootstrapList`. Swapping 1st and 3rd arguments seems to replace correctly

Reply:
Nice catch! Looks like we only want the filtered bootstrap list for `destConfig`.

Reply:
The perils of 1AM coding.Nice catch!